packages/next/src/server/next-server.ts

```diff
@@ -171,6 +174,8 @@ export default class NextNodeServer extends BaseServer<
     res: ServerResponse
   ) => void
 
+  private internalWaitUntil: WaitUntil | undefined
```

Comment:
We've used the word `internal` to refer to things a user shouldn't interact with, not our base implementation. Maybe saying `builtInWaitUntil` is clearer?

Reply:
(we agreed to come back to it later)